lib/gcloud/translate/api.rb

```diff
@@ -196,9 +194,8 @@ def translate *text, to: nil, from: nil, format: nil, cid: nil
       def detect *text
         return nil if text.empty?
         text = Array(text).flatten
-        resp = connection.detect(text)
-        fail ApiError.from_response(resp) unless resp.success?
-        Detection.from_response resp, text
+        resp = service.detect(text)
```

Comment:
The variable is no longer a response object, but is the api object. Can you change the variable name to `gapi` to reflect this?